lib/install.js

```diff
@@ -7,6 +7,7 @@ var debug = require('debug')('strong-pm:install');
 var fs = require('fs');
 var path = require('path');
 var slServiceInstall = require('strong-service-install');
+var dbUpgrade = require('./upgrade-db');
```

Comment:
sort? I prefer to keep names same: `upgrade-db.js` -> `upgradeDb` (or reverse the js file name).